### DISCUSSION TITLE
Change renewal interval?

### DISCUSSION THREAD

**Discussion by GRL570810** [acmesh-official/acme.sh#4632] [Q&A] [answered]
Newbie question. New to acme.sh that is, I've been using win-acme on a Windows hosting server for years, but have just switched to Ubuntu so am learning all the new tools.
In win-acme there was settings json file that allowed you to tweak a number of parameters around the certificate creation and renewal. One of those settings allowed you to adjust the interval between certificate renewals. The  default was 60 days, but I had adjusted that to 75 days as two weeks is heaps of notice to troubleshoot any renewal issues (& I was intending to go to 82 days as frankly a week is plenty enough). Is there an equivalent file in the acme.sh implementation, and if so where is it located as I haven't managed to locate it? Or is there an alternative method I need to use to adjust the renewal interval?
TIA for any suggestions,
Graham

**Comment by dguertin** [comment]
I'm also new to acme.sh and have the same question. I can change the renew interval by editing the acme.sh script and changing DEFAULT_RENEW from 60 to something else, but this is a manual process. All of our servers are provisioned automatically with Ansible, so I'm looking for a config file or something that I can script a custom renew interval in. Does such a thing exist?

**Reply by dguertin** [reply]
In the mean time, you can automate editing the acme.sh script with sed (or in Ansible use the lineinfile module).

**Comment by neilpang** [comment, marked as answer]
when you issue a cert, you can define how many days to renew the cert:

eg: 30 days:

```
acme.sh --issue .......  --days 30
```


you can also use the validity feature:

https://github.com/acmesh-official/acme.sh/wiki/Validity
